### DISCUSSION TITLE
use-form schema, remove commas from array message

### DISCUSSION THREAD

**Discussion by oskar-gmerek** [mantinedev/mantine#1282] [Q&A] [answered]
Hi all,

I have question if it is possible to remove commas in easy way from message in validation when a message is an array.

`const test = ['x', 'y', 'z']`

```
    .superRefine((data, ctx) => {
      if (data.testfield >= something && data.testfield !== otherthing) {
        ctx.addIssue({
          code: z.ZodIssueCode.custom,
          message: `${test}`,
          path: ['testfield'],
        })
      }
```

`The output is: x,y,z`

I'm using more than one library for messages and all of them have translations with dot at the end so output looks bad, like that:

Unwanted output: `Some warning., Some other warning., And the 3rd warning.`

This commas behind dots looks very poor.

Wanted output: `Some warning. Some other warning. And the 3rd warning.` (dots coming from 3rd parties libraries)

It will be great if there is an easy way to sort it out in Mantine as otherwise i will need to change all translations for few languages.

**Thank you in advance!**

**Comment by rtivital** [comment]
Not really sure if that is possible with current implementation, these commas are added by zod. You can figure out how to control errors output with zod or create your own resolver that will manage that (example for zod – https://github.com/mantinedev/mantine/blob/dev/src/mantine-form/src/resolvers/zod-resolver/zod-resolver.ts)

**Reply by oskar-gmerek** [reply]
@rtivital Thanks for advice. Will look at it from zod side.

**Comment by oskar-gmerek** [comment, marked as answer]
If someone in the future will be pulling their hair out how to solve it, probably the easiest solution is simply to use the join method.

```
message: `${test.join(' ')}`,
```
